packages/frontend/web-ui/src/scss/auth/login/_page.scss

```
@@ -0,0 +1,40 @@
+@use '../../base/variables' as var;
+
+.login-form-grid {
+  background-color: var(--cg-text-primary);
+  border-radius: 10px;
+  text-align: center;
+  color: var(--cg-bkg-primary-light);
+  padding: 3rem;
+  margin: 0rem 2rem;
+}
+
+@media (min-width: var.$sm) {
+  .login-form-grid {
+    margin: 0;
+    max-width: 450px;
+  }
+}
+
+.login-title-text-position {
```

I think you want to apply this styles to the login form header, right? What about `login-form-title` instead? Styles I feel it would be more meaningfull candidate name for the class.

```suggestion
.login-form-title {
```

---

Done! I've renamed the CSS class name.